pkg/aggregator/aggregator.go

```diff
@@ -95,6 +96,13 @@ var (
 	aggregatorEvent                            = expvar.Int{}
 	aggregatorHostnameUpdate                   = expvar.Int{}
 
+	tlmFlush = telemetry.NewCounter("aggregator", "flush",
```

Comment:
Improvement idea (feel free to skip it):

To make sure you have the correct number of tags when calling `Inc` or `Add` you can add a wrapper over the generic `Counter`.
```go
type Counter2 struct {
  counter Counter
}

NewCounter2Tags(subsystem, name string, tag1 string, tag2 string, help string) Counter2 { /* ... */ }
func (c *Counter2) Inc(tag1 string, tag2 string) { /* .. */ }
```

You can do it for 0, 1 and 2 arguments for example (If more use `Counter`).

It requires  extra code but you have 54 occurences of telemetry.NewCounter.

Reply:
This is not really idiomatic and to be honest I don't think this is a good idea.
If a developer adds a metric and calls any of its method without the good amount of tags, it'll crash during the development phase and it will be extremely visible (in unit tests, QA, etc.)
I think that a solution to implement what you're mentioning would be to store the amount of tags necessary in the `NewCounter` method and to ensure that the amount of tags provided on the call of `Inc`, `Set`, etc. is correct, however, it would have a performance impact and these methods are called in hot path.

Reply:
As most methods are used as I thought the idea is less interesting.